src/main/java/org/jenkinsci/plugins/workflow/steps/StepDescriptor.java

```diff
@@ -93,6 +104,79 @@ public boolean isAdvanced() {
     }
 
     /**
+     * Some steps, such as {@code CoreStep} or {@code GenericSCMStep} can take
```

Comment:
Can we do that in a follow up change? There's some additional complication around how to migrate `svn` and `git` steps, too.